pkg/scalers/pulsar_scaler.go

```diff
@@ -28,6 +28,7 @@ type pulsarScaler struct {
 type pulsarMetadata struct {
 	adminURL                      string
 	topic                         string
+	isPartitionedTopic            bool
```

Comment:
this property is not read anywhere, what is the purpose then?

Reply:
I misunderstood the data model. I'll remove this field since it is not necessary.